packages/react-composites/tests/browser/call/hermetic/CallReadiness.test.ts

```diff
@@ -6,21 +6,19 @@ import { expect, Page } from '@playwright/test';
 import {
   dataUiId,
   isTestProfileMobile,
-  isTestProfileStableFlavor,
   pageClick,
   stableScreenshot,
   waitForPageFontsLoaded,
   waitForSelector
 } from '../../common/utils';
 import type { MockCallAdapterState } from '../../../common';
 
+/* @conditional-compile-remove(call-readiness) */
```

Comment:
Nice! How do you figure out all these feature tags? I can imagine it requires lots of code reading (But we don't have to do it any more after your PR!)

Reply:
Yeah, just my best guess followed by asking the feature owners.